### PR TITLE
(gitignore) gitignore only stuff included in project

### PR DESCRIPTION
**Pull request by anik120**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,35 +1,16 @@
...
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
 !vendor/**/zz_generated.*
 
-# editor and IDE paraphernalia
-.idea
-*.swp
-*.swo
-*~
```
Comment (tmshort) [inline .gitignore:32]: Removing this can be an issue with EMACS users (such as myself).

**Review by tmshort — COMMENTED** [review]
Why are these files being removed from gitignore? There is no reason to check any of these in, so it's not unreasonable to have them in the file.

File: .gitignore [hunk 2]
```diff
@@ -1,35 +1,16 @@
...
-dist/**
 operator-controller.yaml
-install.sh
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
 !vendor/**/zz_generated.*
 
-# editor and IDE paraphernalia
-.idea
```
Comment (joelanford) [inline .gitignore:29]: I use GoLand, which creates this directory in the repo root. Please leave this in.

File: .gitignore [hunk 3]
```diff
@@ -1,35 +1,16 @@
...
-*.swp
-*.swo
-*~
+# Dependency directories
+vendor/
 
-# TODO dfranz remove this line and the bin folder when tools binaries are moved to their own folder
-!bin/.dockerignore
```
Comment (joelanford) [inline .gitignore:35]: Are the tool binaries in separate directories now? If not, this (and the `.dockerignore` file) should stay.

File: .gitignore [hunk 4]
```diff
@@ -1,35 +1,16 @@
...
 *.out
 
 # Release output
-dist/**
 operator-controller.yaml
-install.sh
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
 !vendor/**/zz_generated.*
```
Comment (joelanford) [inline .gitignore:12]: We're not vendoring anything, so I think we want to remove this as well?

File: .gitignore [hunk 5]
```diff
@@ -1,35 +1,16 @@
...
+bin/
 
-# Output of the go coverage tool, specifically when used with LiteIDE
+# Output of the go coverage tool
 *.out
 
 # Release output
-dist/**
 operator-controller.yaml
-install.sh
```
Comment (joelanford) [inline .gitignore:22]: Don't get generate `install.sh` as part of `make run|e2e|release`? And goreleaser creates the `dist` directory when we build binaries and run the release. So I think these lines need to stay.

**Pull request closed without merging**